docs/source/index.rst

```diff
@@ -16,6 +16,23 @@ Pseudopeople is currently in an alpha release.
 Things are still in flux!
 If you notice any issues, please let us know `on GitHub <https://github.com/ihmeuw/pseudopeople/issues>`_.
 
+Introduction
+------------
+
+The University of Washington IHME Simulation Science Team is excited to introduce pseudopeople, the Python package that simplifies Probabilistic Record Linkage (PRL) research and development.
+This package generates large-scale, simulated population data according to specifications by the user, to replicate a range of complexities of real applications of probabilistic record linkage software.
+With sensitive data often required for PRL, accessing and testing new methods and software has been a challenge - until now.
+Our innovative approach creates realistic, simulated data including name, address, and date of birth, without compromising privacy. 
+
+Our work builds on the success of previous data synthesis projects, such as
+`FEBRL <http://users.cecs.anu.edu.au/~Peter.Christen/Febrl/febrl-0.3/febrldoc-0.3/manual.html>`_,
+`GeCO <https://dl.acm.org/doi/10.1145/2505515.2508207>`_,
+and SOG,
```

Comment:
There is a reference for this somewhere, SOG stands for Synthetic Occupancy Generator, I think.

Reply:
I know, but I couldn't find a reference that wasn't on a random website!